django_restql/mixins.py

```diff
@@ -564,4 +565,34 @@ def update(self, instance, validated_data):
             fields["many_to"]["one_related"]
         )
 
-        return super().update(instance, validated_data)
\ No newline at end of file
+        return super().update(instance, validated_data)
+
+
+class RestQLViewMixin(object):
+    @property
+    def restql_query(self):
+        if "query" in self.request.query_params.keys():
+            try:
+                return Parser(self.request.query_params["query"]).get_parsed()
+            except SyntaxError:
+                pass
+        return None
+
+    def get_restql_orm_mapping(self):
+        if hasattr(self, "restql_orm_mapping"):
+            return self.restql_orm_mapping
+        elif (
+            self.get_serializer_class()
+            and hasattr(self.get_serializer_class(), "Meta")
+            and hasattr(self.get_serializer_class().Meta, "restql_orm_mapping")
+        ):
+            return self.get_serializer_class().Meta.restql_orm_mapping
+
+        return None
+
+    def get_restql_queryset(self, queryset):
+        if self.restql_query is not None:
+            restql_keys = Parser(self.request.query_params["query"]).get_dict()
+            mapping = self.get_restql_orm_mapping()
```

Comment:
We originally tried to do this, but ran into an issue with the `super()` call. Normally mixins are put before the base class, but if we use `super()`, we'll likely need to specify it after. I'll try it out and see what I get.

Reply:
Works great! We'll debug our usage ourself to figure out why we ran into that issue. It may be that we were sometimes not defining `queryset` and only overriding `get_queryset`. 🙂